bridge/engine/EngineStreamDirector.h

```diff
@@ -856,10 +859,16 @@ class EngineStreamDirector
     /**
      * Checks for ssrcs belonging to a default level if there are participants without pin targets.
      */
-    inline bool isUnpinnedQualityUsed(const uint32_t quality,
+    inline bool isUnpinnedQualityUsed(const size_t quality,
+        const size_t highestAvailableQuality,
         const size_t senderEndpointIdHash,
         const bool isSenderInLastNList)
     {
+        if (quality == highQuality)
```

Comment:
```suggestion
        if (quality == highestAvailableQuality)
```

If the quality is equal to `highestAvailableQuality` then quality is the pinned qualitty

Reply:
`if (quality == highestAvailableQuality)` this is not right, as it will result in some medium and low streams considered "unused for unpinned".  The change you propose basically says: _"we never stream highest available quality to unpinned"_, which is not true. We stream say, _low_, to unpinned, and highest available quality might be low as well (as it is indeed for the screen sharing).